src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -590,7 +590,7 @@ private function addServiceInclude(string $cId, Definition $definition): string
                 }
             }
 
-            foreach ($this->serviceCalls as $id => list($callCount, $behavior)) {
+            foreach ($this->serviceCalls as $id => [$callCount, $behavior]) {
```

Comment:
please revert the CS changes, they might create merge conflicts and are unrelated

Reply:
huh? how did that happen... I didn't change it on purpose 🤔